samples/charts/category-chart/axis-range/App.razor

```diff
@@ -0,0 +1,43 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igOptions">
```

Comment:
add a slider with min = 0, max = 100 and step = 10 to control the YAxisMinimumValue  
add a slider with min = 100 , max = 200 and step = 10 to control the YAxisMaximumValue